crates/nu-command/src/filters/find.rs

```diff
@@ -69,8 +69,8 @@ impl Command for Find {
                 result: None,
             },
             Example {
-                description: "Search and highlight text for a term in a string",
-                example: r#"'Cargo.toml' | find toml"#,
+                description: "Search and highlight text for a term in a string. Note that regular search is case insensitive",
+                example: r#"'Cargo.toml' | find Cargo"#,
```

Comment:
if you're demonstrating case insensitive search, wouldn't it be better to have this as the example?
```suggestion
                example: r#"'Cargo.toml' | find cargo"#,
```

Reply:
Ah yes, my bad.